src/rgw/rgw_http_client.h

```diff
@@ -148,6 +148,83 @@ class RGWHTTPHeadersCollector : public RGWHTTPClient {
 };
 
 
+#if 1
```

Comment:
Ok, what is broken here?  FIXME sounds like something is...not fixed?  Is it fixed later?